src/search/FindInFiles.js

```diff
@@ -657,8 +665,8 @@ define(function (require, exports, module) {
      * Inform node that the list of files has changed.
      * @param {array} fileList The list of files that changed.
      */
-    function filesChanged(fileList) {
-        if (FindUtils.isNodeSearchDisabled() || fileList.length === 0) {
+    function filesChanged(fileList) {        
+        if (FindUtils.isNodeSearchDisabled() || !fileList || fileList.length === 0) {
```

Comment:
Ugh, sorry about that @zaggino, I automatically assumed that `fileList` would never be an empty array just undefined. Again, sorry.